config/prison_data_production.yml

```diff
@@ -100,6 +100,9 @@ Cardiff:
   - 2015-02-04
   - 2015-03-04
   - 2015-03-30
+  - 2015-04-03
+  - 2015-04-06
```

Comment:
This completely removes bookings for the entire day. But you're changes state "Monday 6th April – 1.30pm Visit ONLY". If that's the case, you should remove this line and add a [new line to the slot_anomalies](https://github.com/ministryofjustice/prison-visits/blob/master/README.md#slot-anomalies).